### PR TITLE
[homework_11032023]: Add the second homework

### PR DESCRIPTION
**Pull request by KristinaSergejeva**

### CODE REVIEW

File: src/main/java/lv/acodemy/homework/SecondHomeWork.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
+package lv.acodemy.homework;
+
+import org.w3c.dom.ls.LSOutput;
```
Comment (acodemylv) [inline src/main/java/lv/acodemy/homework/SecondHomeWork.java]: Wrong import, you don't need it here.

File: src/main/java/lv/acodemy/homework/SecondHomeWork.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            if (countriesOfEurope[i].equals("Latvia")) {
+                System.out.println("Latvia is my motherland");
+            }
+        }
+
+        // 2 Task
+        // Sometimes it pays to stay in bed on Monday, rather than spending the rest of the week debugging Monday’s code.
+
+        String[] arrayWord = {"Sometimes ", "it ", "pays ", "to ", "stay ", "in ", "bed ", "on ", "Monday, ", "rather ", "than ", "spending ", "the ", "rest ", "of ", "the ", "week ", "debugging ", "Monday’s ", "code."};
+        int[] emptyArray = new int[arrayWord.length];
```
Comment (acodemylv) [inline src/main/java/lv/acodemy/homework/SecondHomeWork.java]: emptyArray not used and can be removed

File: src/main/java/lv/acodemy/homework/SecondHomeWork.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        //If salary is 5000 = Your salary is low.
+        //If salary is between 5000 - 10000 = Your salary is average.
+        //If salary is greater 10000 = Your salary is high.
+
+        int salary = 5000;
+        if (salary < 5000) {
+            System.out.println("Your salary is low.");
+        } else if (salary >= 5000 && salary < 10000) {
+            System.out.println("Your salary is average.");
+        } else if (salary < 10000) {
```
Comment (acodemylv) [inline src/main/java/lv/acodemy/homework/SecondHomeWork.java]: If salary is greater 10000 = Your salary is high.

**Review by acodemylv — CHANGES_REQUESTED** [review]

File: src/main/java/lv/acodemy/homework/SecondHomeWork.java (hunk 1 above)
Comment (KristinaSergejeva) [reply src/main/java/lv/acodemy/homework/SecondHomeWork.java]: The code I wrote didn't print. I looked up the code on the Internet to compare where I did a mistake. Therefore appeared this import.

File: src/main/java/lv/acodemy/homework/SecondHomeWork.java (hunk 3 above)
Comment (KristinaSergejeva) [reply src/main/java/lv/acodemy/homework/SecondHomeWork.java]: This is the result that should be and I don't write the code correctly?

**Review by acodemylv — APPROVED** [review]

**Comment by acodemylv** [comment]
Good job!

**Pull request merged**